src/pages/index.vue

```diff
@@ -1,5 +1,168 @@
+import { computed, definePageMeta, useRouter } from "#imports";
+
+const { locale } = useI18n();
+const router = useRouter();
+
+const prefixedRoute = computed(() => {
+	return `/${locale.value}/`;
+});
+
+definePageMeta({
+	title: "pages.home.title",
+});
+</script>
+
 <template>
-	<main>
-		<h1>Home Page</h1>
-	</main>
+	<MainContent>
+		<div class="flex flex-col">
+			<div class="h-2 w-full bg-primary-800"></div>
+
+			<img src="/assets/images/background-court-ship.jpeg" alt="" class="w-full" />
+			<div class="h-2 w-full bg-primary-400"></div>
+			<div
+				class="mx-auto mt-40 flex w-fit flex-col justify-center border-b-4 pb-4 text-center text-5xl text-primary-100 lg:flex-row"
+			>
+				<p class="font-black">
+					Der Wiener Hof
+					<span class="hidden pr-2 lg:inline-block">.</span>
+				</p>
+				<p class="pt-4 font-sans text-3xl font-extralight lg:pt-0 lg:text-5xl">
+					Ein prosopographisches Portal
+				</p>
+			</div>
+		</div>
+		<div class="mb-60 min-h-screen">
+			<div class="mx-auto mt-20 max-w-[32rem] flex-col lg:max-w-[50rem]">
+				<div class="mb-10">
+					<p>
+						VieCPro liefert erstmals
+						<strong>tausende prosopographische Profile</strong>
+						für die überwiegende
+						<strong>Mehrheit des Wiener Hofpersonals</strong>
+						. Gegenwärtig ermöglicht die Datenbank die Recherche des Personals beiderlei Geschlechts
+						wie auch verschiedener sozialer Schichten des Wiener Haupthofstaats von
+						<strong>
+							neun Kaisern und der Teilhofstaate von 14 Kaiserinnen, sechs Kaiserinwitwen, 70
+							Erzherzögen und Erzherzoginnen und weiterer Familienmitglieder
+						</strong>
+						von der Mitte des 17. Jahrhunderts bis zum Ende des 18. Jahrhunderts.
+					</p>
+					<p>
+						Für die
+						<strong>leopoldinische Zeit</strong>
+						umfasst der Personenbestand darüber hinaus erstens zahlreiche Funktionsträger:innen, die
+						am Wiener Hof politisch-administrative Aufgaben und eine ökonomisch-kulturelle Rolle
+						erfüllt haben, zweitens Personen an ausländischen Höfen, die von der Wiener Hofkammer
+						Zahlungen erhalten haben, und drittens in einem verwandtschaftlichen Verhältnis zu
+						Funktionsträger*innen standen.
+					</p>
+					<div class="mx-auto mb-10 text-center">
+						<button
+							class="mt-10 rounded bg-gray-300 px-4 py-2 font-sans uppercase text-gray-600 hover:bg-gray-400 hover:text-gray-800"
+							@click="router.push(prefixedRoute + 'search')"
+						>
+							Zur Datenbank
+						</button>
+					</div>
+					<div>
+						<p class="mb-10 mt-4 text-center text-xl font-black text-primary-100">
+							Diese Datenbank befindet sich im Aufbau!
+						</p>
+					</div>
+				</div>
+				<div class="mt-10 rounded-lg bg-gray-100 p-8">
+					<h1 class="text-left text-3xl font-bold text-primary-100">VieCPro</h1>
+					<p class="mt-2">
+						... ist ein
+						<strong>Gemeinschaftsprodukt zweier Forschungsteams</strong>
+						, die vom
```

Comment:
bei linebreaks wird automatisch ein space eingefügt, das ist teilweise nicht berücksichtigt
```suggestion
						<span>, die vom</span>
```